go.mod

```diff
@@ -64,7 +64,6 @@ require (
 	golang.org/x/crypto v0.0.0-20210506145944-38f3c27a63bf
 	golang.org/x/net v0.0.0-20210505214959-0714010a04ed
 	golang.org/x/sys v0.0.0-20210503173754-0981d6026fa6
-	golang.org/x/text v0.3.6 // indirect
```

Comment:
Did you mean to commit this change to `go.mod`?

Reply:
This change must have been a mistake with the keyboard somehow. Don't remember making this change. I'll undo it.